### PR TITLE
Vpc module

### PR DESCRIPTION
**Pull request by Polina-Gubina**

### CODE REVIEW

**Review by enrrou — APPROVED** [review]

File: plugins/modules/vpc.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+from ansible_collections.opentelekomcloud.cloud.plugins.module_utils.otc import OTCModule
+
+
+class VpcModule(OTCModule):
+    argument_spec = dict(
+        state=dict(default='present', choices=['absent', 'present']),
+        name=dict(required=False),
+        description=dict(required=False),
+        cidr=dict(required=False),
```
Comment (enrrou) [inline plugins/modules/vpc.py:140]: type str is declared in docs, but missing here

File: plugins/modules/vpc.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+
+from ansible_collections.opentelekomcloud.cloud.plugins.module_utils.otc import OTCModule
+
+
+class VpcModule(OTCModule):
+    argument_spec = dict(
+        state=dict(default='present', choices=['absent', 'present']),
+        name=dict(required=False),
+        description=dict(required=False),
```
Comment (enrrou) [inline plugins/modules/vpc.py:139]: type str is declared in docs, but missing here

File: plugins/modules/vpc.py (hunk 1 above)
Comment (enrrou) [inline plugins/modules/vpc.py:140]: type str is declared in docs, but missing here

Comment (enrrou) [inline plugins/modules/vpc.py:140]: type str is declared in docs, but missing here

**Review by outcatcher — APPROVED** [review]

File: plugins/modules/vpc.py (hunk 1 above)
Comment (enrrou) [inline plugins/modules/vpc.py:140]: type str is declared in docs, but missing here

**Review by enrrou — CHANGES_REQUESTED** [review]

**Review by enrrou — APPROVED** [review]

**Pull request merged**